e2e-test/src/tests/analyze.test.js

```diff
@@ -0,0 +1,106 @@
+
+
+describe('download dataset', () => {
+  it('transform table', async () => {
+    await expect(page).toClickXPath(vBtn('Your Datasets'));
+
+    await uploadDataset('tableTest');
+    await expect(page).toClickXPath(vListTile({ title: 'tableTest.csv ', content: 'Dataset ready for analysis.' }) + vBtn('View Data'));
+
+    // click transform table
+    await expect(page).toClickXPath(vListTile({ title: 'Transform Table' }));
+
+    // change normalize to 'Min Max'
+    await expect(page).toClickXPath("//label[@class='v-label theme--light'][contains(text(), 'Min Max')]");
+    await expect(page).toClickXPath(vListTile({ title: 'Download Data' }));
+
+    // verify the value has been changed: hard coded but there might be a better way to do that
+    await page.waitForXPath("//div[@class='vue-recycle-scroller__item-view'][1]//div[@class='column']/div[@class='cell type-sample'][1][contains(text(), '0.249')]");
+    await expect(page).toClickXPath(vListTile({ title: 'Transform Table' }));
+
+    // change transform to 'log 10'
+    await expect(page).toClickXPath("//label[@class='v-label theme--light'][contains(text(), 'Log 10')]");
+
+    // click download data to verify the change
+    await expect(page).toClickXPath(vListTile({ title: 'Download Data' }));
+    await page.waitForXPath("//div[@class='vue-recycle-scroller__item-view'][1]//div[@class='column']/div[@class='cell type-sample'][1][contains(text(), '-0.603')]");
+
+    await expect(page).toClickXPath(vListTile({ title: 'Transform Table' }));
+
+    // change scale to 'Pareto Scaling'
+    await expect(page).toClickXPath("//label[@class='v-label theme--light'][contains(text(), 'Pareto Scaling')]");
+    await expect(page).toClickXPath(vListTile({ title: 'Download Data' }));
+    await page.waitForXPath("//div[@class='vue-recycle-scroller__item-view'][1]//div[@class='column']/div[@class='cell type-sample'][1][contains(text(), '0.393')]");
+  });
+  it('analyze table and select matabolite', async () => {
+    let selectedMetabolites;
+    await expect(page).toClickXPath(vBtn('Your Datasets'));
+
+    await uploadDataset('tableTest');
+    await expect(page).toClickXPath(vListTile({ title: 'tableTest.csv ', content: 'Dataset ready for analysis.' }) + vBtn('View Data'));
+
+    // click analyze table
+    await expect(page).toClickXPath(vListTile({ title: 'Analyze Table' }));
+
+    // click ANOVA
+    await expect(page).toClickXPath(vListTile({ title: 'ANOVA' }));
+
+    // select some metabolites from ANOVA by adding the highlighted Metabolites to the selected set
+    /* eslint prefer-template: "off" */
+    await expect(page).toClickXPath("//*[text()='C26 - C26.Fol']" + vBtn({ title: 'Adds the highlighted Metabolites to the selected set' }));
+    selectedMetabolites = await page.$x("//div[@class='v-input v-input--selection-controls v-input--checkbox v-input--hide-details v-input--is-label-active v-input--is-dirty theme--light']");
+    await expect(selectedMetabolites).toHaveLength(1);
```

Comment:
This is also hard coded value because I know how many check boxes it would select every time.
I had a hard time finding the corresponding xpath for verifying specific check boxes are selected every time.

Reply:
This is perfect :+1: 
Can you add a comment documenting which checkbox should be selected? It would be very helpful if this assertion ever failed to know what metabolite should have been checked.